src/Scripts/Game/Core/EL_Utils.c

```diff
@@ -1,5 +1,76 @@
 class EL_Utils
 {
+	//------------------------------------------------------------------------------------------------
+	//! Returns the Owner of the storage the item is in
+	//! \param item Item Entity to get Owner from
+	//! \return Entity Storage Owner
+	static IEntity GetStorageOwner(notnull IEntity item)
```

Comment:
- [ ] Make issue for after merge of quantity that these additions can probably moved to the inventory utils out of the generalized ones

Reply:
https://github.com/EveronLife/EveronLife/issues/171